src/lib.rs

```diff
@@ -1,10 +1,14 @@
-use anyhow::Result;
+//! Our module that implements the dials research app.`
 
+extern crate core;
```

Comment:
What is this for? We are not `nostd` so does this do anything?

Reply:
No idea why that is there, must have been some super strange auto import